src/backend/_scss/application.scss

```diff
@@ -129,7 +129,7 @@ a {
 
 /* Navbar adjustments */
 .logo {
-  max-height: 45px;
```

Comment:
I think that when both the max-height and max-width of the image are set, then the aspect ratio of the image is maintained. Please correct me if I am wrong. Thanks :)